### PR TITLE
'About Port Forwarding' notification

### PR DESCRIPTION
**Pull request by jim-docker**
Guide the user to the port-forwarding "dashboard" when they've added their first port-forward

https://user-images.githubusercontent.com/40840436/140533507-012cd53c-df0c-4b57-a7fb-bf3a4bffcd22.mov

enhances #2340

### CODE REVIEW

File: src/renderer/components/+network-services/service-port-component.tsx [hunk 1]
```diff
@@ -92,13 +92,21 @@ export class ServicePortComponent extends React.Component<Props> {
 
     this.waiting = true;
 
+    // determine how many port-forwards are already active
+    const { length } = await getPortForwards();
```
Comment (Nokel81) [inline src/renderer/components/+network-services/service-port-component.tsx]: shouldn't this be within the `try`?

**Review by Nokel81 — DISMISSED** [review]
Nit: looks good. I think I agree that introducing a new endpoint just for this is a bit overkill for the time being.

File: src/renderer/components/+network-services/service-port-component.tsx (hunk 1 above)
Comment (jim-docker) [reply src/renderer/components/+network-services/service-port-component.tsx]: good point

File: src/renderer/port-forward/port-forward-notify.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+import React from "react";
+import { portForwardsURL } from "../../common/routes/port-forwards";
+import { Button } from "../components/button";
+import { Notifications, notificationsStore } from "../components/notifications";
+import { navigate } from "../navigation";
+import { getHostedClusterId } from "../utils";
+
+
+export async function aboutPortForwarding() {
```
Comment (chenhunghan) [inline src/renderer/port-forward/port-forward-notify.tsx]: Does this need to be async?

Comment (jim-docker) [reply src/renderer/port-forward/port-forward-notify.tsx]: I wondered that myself. Probably not, thanks

**Review by chenhunghan — DISMISSED** [review]
LGTM

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**